### ISSUE TITLE
The hook DisplayProductPriceBlock is not available in the list of hooks in the latest development version of prestashop (1.7.6.0)

### ISSUE THREAD

**Issue by preoteasa** [PrestaShop/PrestaShop#13510]
**Describe the bug**
The hook displayProductPriceBlock is not available in the list of hooks in the latest development version of Prestashop (1.7.6.0)

**To Reproduce**
1. Go to 'Back office - Design - Positions - Transplant a module'
2. Select for example the module "Currency block"
3. In the list "Transplant to", the hook displayProductPriceBlock is not available.
4. The same in 1.7.3.3 has displayProductPriceBlock in the list of available hooks.

**Additionnal information**
PrestaShop version: 1.7.6.0 (latest github version cloned on 21.04.2019)
PHP version: N/A

**Comment by khouloudbelguith** [comment]
Hi @preoteasa,

I installed a demo module which uses the displayProductPriceBlock hook
[mymodule.zip](https://github.com/PrestaShop/PrestaShop/files/3103087/mymodule.zip)
In BO => Positions page => When I transplant the currency block module => this hook is available.
I tried with PS1.7.5.1 & branch 1.7.6.x.
![image](https://user-images.githubusercontent.com/16067358/56497008-bf403f00-64f3-11e9-8adc-c3f0105217eb.png)
Thanks to check & feedback.

**Comment by preoteasa** [comment]
Thank you for your answer. I did try using some earlier version of prestashop, probably 1.7.3.3 and it worked. However not the latest repository version that I downloaded about one week ago. As you can see in the next screenshot, displayProductPriceBlock in not in the list of hooks.
![prestashop](https://user-images.githubusercontent.com/6583978/56833442-2a34a180-6877-11e9-8159-9dfe02661d83.png)

I also just updated from github, and the result is the same.

**Comment by khouloudbelguith** [comment]
Hi @preoteasa,

Thanks for your feedback.
If you try to install my demo module which uses the displayProductPriceBlock hook, this hook is not displayed also in your list after the install of my demo module?
Thanks to check & feedback.

**Comment by preoteasa** [comment]
Hello,

I tried to install your module on a clean install of the latest development version of Prestashop. After the clean installation, the hook displayProductPriceBlock is still not available for the  module "Currency block". After installing mymodule.zip, the hook becomes available.

Looking at mymodule, I noticed the following: mymodule registers hook displayProductPriceBlock, but implements old hook hookDisplayProductButtons. If the implementation of hookDisplayProductButtons
is 
		public function hookDisplayProductButtons($params){
			return "Test B";
		}
Then nothing is displayed in the product page. If in addition I add:
	public function hookDisplayProductPriceBlock($params)
	{
		return "Test A";
	}

then Test A is displayed as expected.

It seems that there is a problem with this hook, as only the presence of hookDisplayProductButtons would trigger its appearance in the list of hooks, however hookDisplayProductButtons is not executed.
